tests/console/suseconnect_scc.pm

```diff
@@ -49,8 +49,8 @@ sub run {
     if (is_sle) {
         # What has been activated by default
         assert_script_run q[SUSEConnect --list-extensions | grep -e '\(Activated\)'];
-        assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mServer Applications Module\')"';
-        assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'        \e\[1mWeb and Scripting Module\')"';
+        assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mServer Applications Module\')"' unless is_jeos;
```

Comment:
This is not `jeos` specific`, is it a leftover?

Reply:
the PR is still draft. what is missing is `is_jeos && && is_sle('=12-sp5')`

Reply:
It should be without `is_jeos` as there are no JeOS modules.

Reply:
ready for review